tools/parse_module_type.py

```diff
@@ -80,11 +80,28 @@ def parseWebsite():
 
     # ICS modules
     modules_with_type['DB&S'] = kernmodul
+    modules_with_type['NETW1'] = kernmodul
     modules_with_type['NETW2'] = kernmodul
+    modules_with_type['ADS'] = kernmodul
+    modules_with_type['INTROL'] = kernmodul
+    modules_with_type['SPTA'] = kernmodul
+    modules_with_type['SPREN1'] = kernmodul
+    modules_with_type['ISLAB_K'] = kernmodul
+    modules_with_type['ANLS'] = kernmodul
+    modules_with_type['SPTA'] = kernmodul
+    modules_with_type['CRS'] = erweiterungsmodul
+    modules_with_type['DSO'] = kernmodul
+    modules_with_type['PRIVACY1'] = kernmodul
+    modules_with_type['OSA'] = kernmodul
+    modules_with_type['ISM'] = kernmodul
+
+    # only for ics students
+    # modules_with_type['ETHIK'] = kernmodul
```

Comment:
there might be more modules like this.. see #9. We can leave the commented out line in like it is for now. But issue #9 should be addressed to fix this.

Reply:
I haven't checked all I and WI modules, but I will add them for the moment like this, until #9 is implemented.

Reply:
Or HSLU updates the ICS module page and the type can be extracted automatically